zebrad/src/components/mempool/config.rs

```diff
@@ -39,10 +28,18 @@ pub struct Config {
     ///
     /// This corresponds to `mempoolevictionmemoryminutes` from
     /// [ZIP-401](https://zips.z.cash/zip-0401#specification).
-    ///
-    // Note: Durations become a TOML table, so they must be the final item in the config
-    //       We'll replace them with a more user-friendly format in #2847
+    #[serde(with = "humantime_serde")]
     pub eviction_memory_time: Duration,
+
+    /// If the state's best chain tip has reached this height, always enable the mempool,
+    /// regardless of Zebra's sync status.
+    ///
+    /// Set to `None` by default: Zebra always checks the sync status before enabling the mempool.
+    //
+    // TODO:
+    // - allow the mempool to be enabled before the genesis block is committed?
+    //   we could replace `Option` with an enum that has an `AlwaysEnable` variant
+    pub debug_enable_at_height: Option<u32>,
```

Comment:
It's not clear to me why you moved the field to the bottom.

Reply:
There was a `TODO` in line 19 of this file that was saying:
```
// - move debug configs last (needs #2847)
```

So i think we want to have the debug options at the end of each section as the other options are more important. That is the reason i moved, i could revert that part if it is not a good idea anymore, the `TODO` comment may be pretty old.

Reply:
Makes sense, thank you for explaining. I think it's good the way it is.